sonar-plugin/pom.xml

```diff
@@ -3,32 +3,32 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>io.ecocode</groupId>
-    <artifactId>ecocode-javascript-plugin</artifactId>
+    <groupId>org.green-code-initiative</groupId>
+    <artifactId>creedengo-javascript-plugin</artifactId>
     <version>1.5.1-SNAPSHOT</version>
 
     <packaging>sonar-plugin</packaging>
 
-    <name>ecoCode JavaScript plugin</name>
+    <name>Creedengo JavaScript plugin</name>
     <description>Provides rules to reduce the environmental footprint of your JavaScript programs</description>
     <inceptionYear>2023</inceptionYear>
```

Comment:
please change year to update copyright year in all files

Reply:
my answer here: https://github.com/green-code-initiative/creedengo-javascript/pull/56#discussion_r1911427555